openslides/__main__.py

```diff
@@ -104,9 +104,9 @@ def get_parser():
         help='The used settings file. The file is created, if it does not exist.')
     subcommand_start.set_defaults(callback=start)
     subcommand_start.add_argument(
-        '--development',
+        '--local-installation',
         action='store_true',
-        help='Option for development purposes.')
+        help='Store settings and user files in a local directory.')
```

Comment:
No: If you use `manage.py` then OpenSlides will always use the local folder (development, openslides_data ot whatever we call it). If you use `manage.py` with the flag `--local-installation` this does the same. But if you use `openslides start` after you installed a tar.gz version then OpenSlides will install to `~/.config/openslides` and thelike. See #2013.